sonar/src/main/java/com/rbc/sonar/Main.java

```diff
@@ -5,7 +5,9 @@
  */
 public class Main {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9089/coding_rules#rule_key=squid%3AS1118)